packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -174,6 +168,7 @@ export default class ImageEdit implements EditorPlugin {
      * Dispose this plugin
      */
     dispose() {
+        this.removeVisibilityCssTag();
```

Comment:
We also need to remove the editing image DOM element here, and the call to removeVisibilityCssTag() can be part of it.